replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/Syncer.java

```diff
@@ -136,16 +137,17 @@ SyncResponse sync() {
           }
         }
 
-        Optional<ReplicationItem> replicationItem =
-            replicationItemManager.getItem(metadata.getId(), sourceName, destinationName);
-
+        Optional<ReplicationItem> replicationItem = getReplicationItem(metadata.getId());
         try {
           if (metadata.isDeleted() && replicationItem.isPresent()) {
             doDelete(metadata, replicationItem.get());
-          } else if (replicationItem.isPresent() && destination.exists(metadata)) {
-            doUpdate(metadata, replicationItem.get());
           } else {
-            doCreate(metadata, replicationItem);
+            Metadata remoteMetadata = destination.exists(metadata);
+            if (replicationItem.isPresent() && remoteMetadata != null) {
```

Comment:
One thought for the future: a replication log or something like that so we can go back and look at the behavior.